scripts/utils.js

```diff
@@ -28,3 +30,48 @@ export function validJSON(string) {
 export function cleanRecord(record) {
   return omit(record, ["id", "schema", "last_modified"]);
 }
+
+export function recordField(displayField, record) {
```

Comment:
How about renaming this and updating the signature to something more meaningful like `renderDisplayField(record, displayField)`?